stdlib/public/runtime/EnvironmentVariables.cpp

```diff
@@ -187,9 +191,32 @@ extern "C" char **_environ;
 #endif
 #endif
 
+#if defined(__ANDROID__)
+// On Android, also try loading runtime debug env variables from system props.
+static void platformInitialize(void *context) {
+  (void)context;
+#define SYSPROP_PREFIX "debug.swift.runtime."
```

Comment:
Is there a standard way to name properties? Should this perhaps be reverse domain'ed as: `org.swift.runtime.debug.`?